wasatch/DeviceFinderUSB.py

```diff
@@ -1,15 +1,16 @@
 import re
+import usb
 import logging
 import platform
 from functools import partial
+
 if platform.system() == "Darwin":
     # would like to include in Mac stub
     # * imports must be at the module level though
     from ctypes import *
     from CoreFoundation import *
-
-import usb
-import usb.backend.libusb0 as libusb0
+else:
```

Comment:
instead of boolean `self.use_libusb0`, why not store the desired backend in global var `backend` ?

For example
```
if platform.system() == "Darwin":
    import usb.backend.libusb1 as backend
else:
    import usb.backend.libusb0 as backend
```